cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/CgmesDcConversion.java

```diff
@@ -96,16 +96,38 @@ private void convert(List<HvdcConverter> converters, List<String> dcLineSegments
     }
 
     private void convert(String acDcConverterIdEnd1, String acDcConverterIdEnd2, String dcLineSegmentId) {
+        convert(acDcConverterIdEnd1, acDcConverterIdEnd2, dcLineSegmentId, false);
+    }
+
+    private void convert(String acDcConverterIdEnd1, String acDcConverterIdEnd2, String dcLineSegmentId, boolean isDuplicated) {
         if (!convertCommonData(acDcConverterIdEnd1, acDcConverterIdEnd2, dcLineSegmentId)) {
             return;
         }
         this.r = computeR(this.dcLineSegment);
 
+        // After collecting all commomData the id can be modified to guarantee uniqueness
+        if (isDuplicated) {
+            defineACopyOfThePropertyBagWithUniqueId(dcLineSegment, dcLineSegmentId);
+        }
+
         if (createHvdc()) {
             setCommonDataUsed();
         }
     }
 
+    private void defineACopyOfThePropertyBagWithUniqueId(PropertyBag dcLineSegment, String dcLineSegmentId) {
+
+        // Create a copy of the property bag and change the id to guarantee uniqueness
+        PropertyBag dcLineSegmentCopy = dcLineSegment.copy();
+
+        String completeId = dcLineSegmentCopy.get("DCLineSegment");
+        String uniqueId = dcLineSegmentId + "-1";
```

Comment:
I would rather have a `#0` since this is the "normal" nomenclature for duplicated IDs.